e2e/pagerank_online_test/test_online_double_call_set/test.yml

```diff
@@ -0,0 +1,15 @@
+- query: >-
```

Comment:
why are we only testing count?

Reply:
Algorithm is stochastic, therefore we are unable to test the real output.